### ISSUE TITLE
Accessor vertex count vs Draco mesh vertex count

### ISSUE THREAD

**Issue by alexey-pelykh** [KhronosGroup/glTF-Blender-IO#1326]
**Describe the bug**
When loading GLB exported with Draco compression from Blender, Google Filament complains:
```
The glTF accessor wants 187 vertices, but the decoded Draco mesh has 204 vertices.
```
And accessor value is indeed incorrect.

**To Reproduce**
1. Open the attached file
2. Export GLTF with Draco compression
3. Check the number of vertices in the accessor vs the number of vertices in the Draco mesh

**Expected behavior**
Matching number of vertices in GLTF accessor and Draco mesh

**Screenshots**
![image](https://user-images.githubusercontent.com/345102/107281302-2715a000-6a62-11eb-8d93-cfdb110ec04b.png)

**.blend file/ .gltf**
[issue.blend.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/5948831/issue.blend.zip)

Pure Draco data decompressed gives this obj file:
[issue.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/5946875/issue.zip)

**Version**
 - OS: macOS
 - Blender Version: 2.90 or 2.91.2

**Comment by alexey-pelykh** [comment]
Important note: it seems not to be related to Draco compression, even without it, vertex count in accessor vs vertex count in data differs.

**Comment by emackey** [comment]
I'm not able to reproduce this.  I exported to a Draco-compressed file from Blender 2.92 Beta (last night's build), and I'm able to view it in Babylon, ThreeJS, and Filament 1.9.10.  The model appears to be just the box with rounded corners, no extra appendage as shown in the screenshot above.

**Comment by alexey-pelykh** [comment]
@emackey that's great news, I've just tried out 2.92-beta and indeed export files differ in the part that causes the issue: new one claims to have 214 vertices while old one claimed to have 198. Does 2.91.2 produce the same result as on image on Filament native (I'm using macOS) gltfViewer?

**Comment by emackey** [comment]
Confirmed, this is broken in 2.91 and fixed in 2.92.  I think it's safe to close this.
